client/db.go

```diff
@@ -90,7 +89,14 @@ func (kv *KeyValue) ValueBytes() []byte {
 // ValueInt returns the value as an int64. This method will panic if the
```

Comment:
Ah, OK, so the interpretation of "value's type" has changed. This is a little unclear since `Value` is still exposed as an `interface{}`.